chromium_src/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate_factory.cc

```diff
@@ -0,0 +1,8 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/browsing_data/brave_browsing_data_remover_delegate.h"
+
+#include "../../../../../chrome/browser/browsing_data/chrome_browsing_data_remover_delegate_factory.cc"  // NOLINT
```

Comment:
I recall it is no longer needed to add NOLINT for this case?

Reply:
Yes, linter doesn't catch this anymore.
Deleted.